fpga/lib/core.cpp

```diff
@@ -56,6 +56,28 @@ std::list<IpIdentifier> CoreFactory::parseIpIdentifier(json_t *json_ips) {
   return allIps;
 }
 
+std::list<IpIdentifier>
+CoreFactory::filterIps(std::list<IpIdentifier> allIps,
+                       std::list<std::string> ignored_ip_names) {
+  std::list<IpIdentifier> filteredIps;
+
+  std::set<std::string> ignored_ip_set(ignored_ip_names.begin(),
+                                       ignored_ip_names.end());
+  for (const auto &ip : allIps) {
+    const auto &ipName = ip.getName();
+
+    if (ignored_ip_set.find(ipName) != ignored_ip_set.end()) {
+      CoreFactory::getStaticLogger()->warn(
+          "Ignoring Ip {} (explicitly on ignorelist)", ipName);
+    } else {
+      filteredIps.push_back(
+          ip); // Or use `filteredIps.emplace_back(std::move(ip));` if moving is preferred.
```

Comment:
doesn't really matter because this is not performance critical. But please remove the comment, because it adds no information :)